circle.yml

```diff
@@ -0,0 +1,19 @@
+machine:
+  services:
+    - docker
+
+test:
+  override:
+    - echo 0
```

Comment:
Consider adding a comment here to explain why we're echoing `0` rather than running real tests.